diff --git a/submodules/CMakeLists.txt b/submodules/CMakeLists.txt
index ed50dbb..934e6ac 100644
--- a/submodules/CMakeLists.txt
+++ b/submodules/CMakeLists.txt
@@ -2,17 +2,6 @@ include(FetchContent)
 include(ProcessorCount)
 ProcessorCount(PROCESSOR_COUNT)
 
-if(CMAKE_HOST_SYSTEM_NAME STREQUAL "Linux")
-  # Read the os-release file
-  file(READ "/etc/os-release" OS_RELEASE)
-
-  # Extract distribution name
-  string(REGEX MATCH "NAME=\"([^\"]+)\"" _ ${OS_RELEASE})
-  string(TOLOWER "${CMAKE_MATCH_1}" DISTRO_NAME)
-
-  message(STATUS "Linux Distribution: ${DISTRO_NAME}")
-endif()
-
 set(MODULES_DIR "${CMAKE_BINARY_DIR}/.deps")
 file(MAKE_DIRECTORY "${MODULES_DIR}")
 
@@ -27,7 +16,6 @@ endif()
 
 set(MODULES_INSTALL_PREFIX "${MODULES_DIR}/install")
 
-find_program(_GIT_EXE git REQUIRED)
 
 
 function(find_and_replace file_name find_what replace_with)
@@ -180,12 +168,6 @@ else()
   checkout_submodule_branch(${MODULE_GRPC} "https://github.com/grpc/grpc"
                             "v1.70.1" "${MODULES_DIR}/grpc/build-release")
 
-  string(FIND "${DISTRO_NAME}" "alpine" POSITION)
-  if(POSITION GREATER -1)
-    find_and_replace("${MODULES_DIR}/grpc/third_party/re2/util/pcre.h"
-                     "mutable int32_t" "mutable int")
-  endif()
-
   if(APPLE)
     # Fix wrong detection of fdopen on macOS
     find_and_replace("${MODULES_DIR}/grpc/third_party/zlib/zutil.h"
